packages/apputils-extension/src/shortcuts.tsx

```diff
@@ -161,15 +163,23 @@ export function displayShortcuts(options: IOptions) {
     if (groupedBindings.has(d)) {
       bindingTable.push(
         groupedBindings.get(d)!.map(b => (
-          <tr className={SHORTCUT_TABLE_ROW_CLASS} key={b.command}>
+          <tr
+            className={SHORTCUT_TABLE_ROW_CLASS}
+            key={`${b.command}-${b.keys.join('-')}`}
```

Comment:
I got a duplicate key warning when testing it locally with two shortcuts for the same command if both start with a keystroke with modifier, such as `Ctrl X` and `Ctrl Y`. This is because there will be a space which will terminate the key prematurely (both would then have a key `plugin:command-Ctrl`). This should solve the issue:

```suggestion
            key={`${b.command}-${b.keys.join('-').replace(' ', '_')}`}
```